contributing/code/tests.rst

```diff
@@ -3,54 +3,32 @@
 Running Symfony Tests
 =====================
 
-Before submitting a :doc:`patch <patches>` for inclusion, you need to run the
-Symfony test suite to check that you have not broken anything.
+The Symfony project uses a third-party service which automatically runs tests
+for any submitted :doc:`patch <patches>`. If the new code breaks any test,
+you'll be notified in detail about that error.
```

Comment:
you should mention that this will be in the PR status. The contributor will not be notified by email